contracts/src/v0.8/operatorforwarder/test/testhelpers/Deployer.t.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import {Test} from "forge-std/Test.sol";
+
+import {MockReceiver} from "./MockReceiver.sol";
+import {AuthorizedForwarder} from "../../AuthorizedForwarder.sol";
+import {Operator} from "../../Operator.sol";
+import {OperatorFactory} from "../../OperatorFactory.sol";
+import {LinkToken} from "../../../shared/token/ERC677/LinkToken.sol";
+
+abstract contract Deployer is Test {
+  OperatorFactory internal s_factory;
+  LinkToken internal s_link;
+  MockReceiver internal s_mockReceiver;
+
+  address private s_owner = makeAddr("owner");
```

Comment:
I kinda prefer a constant set of owner, alice, bob. That way there's no `s_` and it's all caps. Makes it clearer imo, and doesn't allow to you accidentally overwrite them